### PR TITLE
QMS - LTSS-1 - Measure Overview Page

### PR DESCRIPTION
**Pull request by ailZhou**
### Description

This PR builds out LTSS-1 measure overview page. The content of what's suppose to be on this page is written in the jira ticket, the figma is just for a general idea of what the page might look like,

I also expanded the radio button to have children elements underneath. 
Screenshot 2024-12-03 at 3 52 55 PM

**Note**
1) Children textboxes of the radio button currently does not save any data. Will address in a separate PR.
2) The quality measure table [Edit] buttons do not go anywhere yet. Also will address in a separate PR. 

### Related ticket(s)

CMDCT-4139

---
### How to test

1) Sign into HCBS, any state user
2) Create a QM report
3) On the sidebar menu, select [**Required Measure Results**]. You should see a row for CMIT # 960.
     - The generation for this table is covered by another ticket.
4) Click [**Edit**] and you be taken to the  LTSS-1 measure overview page.
5) Check that the questions matches what is written in the jira ticket: CMDCT-4139.
6) You should see a page that looks like this
localhost_3000_report_QM_PR_2pir7juQ4TtESKqww3nP30icY6A

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/cmit.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { CMIT, DeliverySystem, DataSource, MeasureSpecification } from "types";
```
Comment (ailZhou) [inline services/ui-src/src/cmit.ts:1]: Brendan said it was fine for me to move it here for now if it keeps things moving. We're still figuring out how we want the report to access this data and where

File: services/app-api/forms/qm.ts [hunk 2]
```diff
@@ -186,24 +187,19 @@ export const qmReportTemplate: ReportTemplate = {
...
-        stratified: false,
-        measureTemplate: MeasureTemplateName.StandardMeasure,
-      },
     ],
   },
   measureTemplates: {
-    [MeasureTemplateName.StandardMeasure]: {
-      id: "req-measure-report",
-      title: "Example Measure",
+    [MeasureTemplateName["LTSS-1"]]: {
```
Comment (angelaco11) [inline services/app-api/forms/qm.ts:193]: beutiful 🥲

File: services/app-api/forms/qm.ts [hunk 3]
```diff
@@ -273,5 +309,5 @@ export const qmReportTemplate: ReportTemplate = {
       sidebar: false,
       elements: [],
     },
-  },
+  } as Record<MeasureTemplateName, MeasurePageTemplate>,
```
Comment (angelaco11) [inline services/app-api/forms/qm.ts:312]: oh is this how you fixed that really annoying ts error?! this is so much smarter than whatever i was trying to do 😂

**Review by angelaco11 — APPROVED** [review]
Looks great! Thanks for working with me on this. I tested your changes on cloudfront and the measure details are as expected.

**Review by ajaitasaini — APPROVED** [review]

**Pull request merged**